### ISSUE TITLE
js error

### ISSUE THREAD

**Issue by gatspy** [arl/statsviz#95]
```
plot.js:215 
        
    Uncaught ReferenceError: tippy is not defined
    at Plot._installHeatmapTooltip (plot.js:215:26)
    at Plot.createElement (plot.js:203:18)
    at attachPlots (app.js:100:14)
    at ws.onmessage (app.js:57:13)
```

![image](https://user-images.githubusercontent.com/804446/199422118-9115f876-b995-4cf7-ac40-a49f1d308dc1.png)

**Comment by arl** [comment]
Hi.
Can you share the Go code you're running so we can reproduce please.
Thank you

**Comment by arl** [comment]
Fixed by #106 and released in v0.5.2
